test/testhelpers.sh

```diff
@@ -161,6 +161,21 @@ setup() {
   }
   cp "$HOME/.gitconfig" "$HOME/.gitconfig-backup"
 
+  if [[ `git config --system credential.helper | grep osxkeychain` == "osxkeychain" ]]
```

Comment:
Should this be

    if [[ "`git config --system credential.helper | grep osxkeychain`" == osxkeychain* ]]

to account for cases where someone may have arguments in their credential.helper? (It's at least possible with cache and store)

Also this makes it a bash, So the shebang should be updated to

    #!/usr/bin/env bash

OR

    if [ "`git config --system credential.helper | grep osxkeychain`" == "osxkeychain" ]

if arguments just aren't EVER going to happen with the osxkeychain